cmd/proxy/main.go

```diff
@@ -44,6 +44,20 @@ func main() {
 
 	logger := athenslog.New(conf.CloudRuntime, logLvl)
 
+	// logrus.Writer() spawns a go routine, so instead of doing that each time
+	// we need to make logrus look like a (standard) log.Logger (and then Close()ing it),
+	// we can do it once in main and have the global log.Logger use it.
+	{
+		logrusWriter := logger.Writer()
```

Comment:
Surely INFO is not the correct level here? The components you speak of generally only write to the log in case of Error, so passing the logs off as INFO seems incorrect.

Reply:
Fair point. I guess I wasn't sure if I was ready to say that everything traced via the standard logger would be an Error. What do you think about Warn as a "this trace might be concerning" compromise?

Reply:
Happy to start with Error. 😄

Reply:
Changed in 18f5e26.